src/ui/public/agg_types/controls/precision.html

```diff
@@ -36,3 +36,21 @@
   </label>
 </div>
 
+<div class="vis-option-item indented">
+  <label>
+    <input type="checkbox"
+           name="useFilter"
+           ng-model="agg.params.useFilter">
+    Enable collar (geo_bounding_box filter)
+  </label>
+</div>
+<div class="vis-option-item indented" ng-if="agg.params.useFilter">
+  <div class="form-group">
+    <label>
+      Collar Scale <kbn-info info="Set the size of the geo_bounding_box filter applied to the map bounds. Small values could result in excessive fetches during map panning. Large values could result in trimmed results."></kbn-info>
+    </label>
+    <input type="number" class="form-control"
```

Comment:
I'm not sure if we even need to make this a user-facing setting. I would go for a good default (1.5 is fine). If it proves to be a problem in the wild, let's add it in. So I would just remove this option altogether.

If you think this is important, I'd rework it a little:
- ensure the UI control does not allow to select a value outside the the valid 1-5 range (the kui-framework has a number picker). use something like https://github.com/thomasneirynck/kibana/blob/b86f97ce0f83c50b8737c3cef75773f54d4df64a/src/core_plugins/kbn_vislib_vis_types/public/controls/point_series/value_axes.html#L229-L229
- read the default-value consitently from `this._getGeohashAgg().type.params[2].default` instead of duplicating its declaration in multiple locations.